packages/apollo/README.md

```diff
@@ -228,6 +228,8 @@ OPTIONS
 
   --tagName=tagName                      Name of the template literal tag used to identify template literals containing
                                          GraphQL queries in Javascript/Typescript code
+                                         
+  --target=format                        Format of schema to be downloaded. Currenlty supporting SDL & JSON
```

Comment:
Might make sense to add `[default: JSON]` at the beginning of the description.

Reply:
@FabHof the default filename is already in the argument section here https://github.com/apollographql/apollo-tooling/pull/1470/files#diff-01b0f6fd12a8adc09aea18cfc07f77a3R201